chess-AI/train.py

```diff
@@ -128,7 +129,7 @@ def create_dataset(games, move_approximator, val_approximator=None, show_dash=Fa
     return dataset
 
 
-def train_on_dataset(dataset, nnet, options, show_dash=False):
+def train_on_dataset(dataset, nnet, options, iteration, save=True, show_dash=False):
```

Comment:
I think `save` should be a field in `options` (and correspondingly the params/ai_io/dashboard). don't worry about updating the dashboard tho

Reply:
the value of save varies between calls, its just internal for making it so MCTS training can control whether the model gets saved or not. save_freq is where the user input happens